live-streaming/live-stream-best-practices.md

```diff
@@ -38,49 +31,84 @@ After this, take network connection into consideration - if you or your users st
-| 720p     | (60 fps)  | 60 fps 2250-6000 Kbps      | 44,1 kHz          | 128 Kbps      |
-| 1080p    | (30 fps)  | 25-30 fps 3000-6000 Kbps   | 44,1 kHz          | 128 Kbps      |
-| 1080p    | (60 fps)  | 60 fps 4500-9000 Kbps      | 44,1 kHz          | 128 Kbps      |
-| 2160(4K) | (30 fps)  | 25-30 fps 13000-34000 Kbps | 44,1 kHz          | 192 Kbps      |
-| 2160(4K) | (60 fps)  | 60 fps 20000-51000 Kbps    | 44,1 kHz          | 192 Kbps      |
+If reconnection does not happen within 1 minute, api.video considers the live stream as expired. Any new connection after this is treated as a new live stream event with a new DVR and a new recording.
+
+## Recommendations
+
+<Callout pad="2" type="info">
+When [restreaming](/live-streaming/restreams), you should adapt the original live stream to fit the constraints of the restream destination's server requirements. api.video forwards the originating stream directly to the restream destination as received, without any modification.
+</Callout>
+
+- Bitrate encoding: CBR  
+- Keyframe Interval: 2 seconds
+
+| Quality           | Framerate | Video bitrate     | Audio sample rate | Audio Bitrate |
+| ----------------- | --------- | ----------------- | ----------------- | ------------- |
+| 240p              | 25-30 fps | 300-700 Kbps      | 48 kHz            | 64 Kbps       |
+| 360p              | 25-30 fps | 400-1000 Kbps     | 48 kHz            | 128 Kbps      |
+| 480p              | 25-30 fps | 500-2000 Kbps     | 48 kHz            | 128 Kbps      |
+| 720p (30 fps)     | 25-30 fps | 1500-4000 Kbps    | 48 kHz            | 128 Kbps      |
+| 720p (60 fps)     | 50-60 fps | 2250-6000 Kbps    | 48 kHz            | 128 Kbps      |
+| 1080p (30 fps)    | 25-30 fps | 3000-6000 Kbps    | 48 kHz            | 128 Kbps      |
+| 1080p (60 fps)    | 50-60 fps | 4500-9000 Kbps    | 48 kHz            | 128 Kbps      |
+| 2160p (30 fps)    | 25-30 fps | 13000-34000 Kbps  | 48 kHz            | 192 Kbps      |
+| 2160p (60 fps)    | 50-60 fps | 20000-51000 Kbps  | 48 kHz            | 192 Kbps      |
 
 ## Limitations
 
-<Callout pad="2" type="warning">
+### Live stream recording
+
+Live streams are recorded automatically. The length of each recording is fixed at 12 hours. When a recording reaches this length, a new recording is started.
+
+### DVR
+
+When a live stream is ongoing, viewers can replay earlier content with the DVR feature.
+
+* The available duration for DVR is 1 hour.
+* DVR is only available for live streams that are actively broadcasting.
+* When a live stream is completed, api.video stores the recording for as long as the duration the stream, but **at most for 1 hour**. For example, if the duration of the live stream is 15 minutes, then the recording is available for 15 minutes after the stream is completed. Even if the stream is longer than 1 hour, the recording is only kept for 1 hour. The cached DVR recording is subsequently deleted.
```

Comment:
It is not the `recording` here but the `DVR`

Reply:
Good catch! Fixing ⌛